### ISSUE TITLE
Add overloaded method for SecretKeyRingEditor.revokeUserId() with selection strategy as parameter

### ISSUE THREAD

**Issue by vanitasvitae** [vanitasvitae/pgpainless#218]
@vanitasvitae I have another code:
```kotlin
            val editor = PGPainless.modifyKeyRing(secretKeyRing)
            ....
            val selectionStrategy = SelectUserId.or(
              SelectUserId.containsEmailAddress(uid.email),
              SelectUserId.exactMatch(uid.email)
            )
            editor.deleteUserId(selectionStrategy, unprotectedProtector)
```
As you recommended, I've tried to change `deleteUserId()` to `revokeUserId()`, however there is no overloaded method `revokeUserId()` which takes selection strategy as parameter. Please add such one.

_Originally posted by @IvanPizhenko in https://github.com/pgpainless/pgpainless/issues/216#issuecomment-977199320_

**Comment by vanitasvitae** [comment]
This has been implemented in version 1.0.0-rc5 which was just released.

**Comment by IvanPizhenko** [comment]
Thank you, I will try it.
